src/Aspire.Hosting.AWS.DynamoDBLocal/DynamoDBLocalResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Sockets;
+using System.Text.Json;
+using Amazon.DynamoDBv2;
+using Amazon.Runtime;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.AWS.DynamoDBLocal;
+
+namespace Aspire.Hosting;
+
+public static class DynamoDBLocalResourceBuilderExtensions
+{
+    /// <summary>
+    /// Add an instance of DynamoDB local. This is a container pulled from Amazon ECR public gallery.
+    /// Projects that use DynamoDB local can get a reference to the instance using the WithAWSDynamoDBLocalReference method.
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <param name="name">The name of the resource.</param>
+    /// <param name="image">Optional: default value is "public.ecr.aws/aws-dynamodb-local/aws-dynamodb-local"</param>
+    /// <param name="tag">Optional: default value is "latest"</param>
+    /// <param name="disableDynamoDBLocalTelemetry">Optional: default to false. It controls setting the DynamoDB local DDB_LOCAL_TELEMETRY environment variable.</param>
+    /// <returns></returns>
+    /// <exception cref="DistributedApplicationException"></exception>
+    public static IResourceBuilder<IDynamoDBLocalResource> AddAWSDynamoDBLocal(this IDistributedApplicationBuilder builder,
+        string name, string image = "public.ecr.aws/aws-dynamodb-local/aws-dynamodb-local", string tag = "latest", bool disableDynamoDBLocalTelemetry = false)
+    {
+        var container = new DynamoDBLocalResource(name, disableDynamoDBLocalTelemetry);
+        var containerBuilder = builder.AddResource(container)
+                  .WithAnnotation(new ManifestPublishingCallbackAnnotation(SkipWriteToManifest))
```

Comment:
You can use `ManifestPublishingCallbackAnnotation.Ignore`

Reply:
Or ExcludeFromManifest()

Reply:
Nice, switch to use `ExcludeFromManifest`.